docs/installation/server-config/integrate-ksql-with-confluent-control-center.rst

```diff
@@ -0,0 +1,105 @@
+.. _integrate-ksql-with-confluent-control-center:
+
+Integrate KSQL with |c3|
+########################
+
+You can develop event streaming applications by using the KSQL user interface
+provided by |c3|. In |c3-short|, you can create Kafka topics and develop
+persistent queries in the KSQL query editor. When you install |cp|, KSQL Server
+is integrated with |c3-short| by default, and you can configure |c3-short| to
+interact with other KSQL Server instances that run on separate hosts.
+
+.. image:: ../../../../images/ksql-interface-create-stream.png
+     :width: 600px
+     :align: center
+     :alt: Screenshot of the KSQL Create Stream interface in Confluent Control Center.
+
+Configuration Settings for KSQL and |c3|
+****************************************
+
+Set up the integration between KSQL and |c3| by assigning configuration
+properties in the KSQL Server configuration file at 
+``<path-to-confluent>/etc/ksql/ksql-server.properties`` and the |c3-short|
+configuration file at 
+``<path-to-confluent>/etc/confluent-control-center/control-center.properties``.
+
+* In the KSQL Server configuration file, set the :ref:`ksql-listeners` property
+  to the IP address of the REST API endpoint for KSQL Server. Typical values
+  are ``http://0.0.0.0:8088`` and ``http://localhost:8088``.
+* In the |c3-short| configuration file, set the ``confluent.controlcenter.ksql.url``
+  property to the KSQL Server hostname. This setting specifies how |c3-short|

```

Comment:
> set the ``confluent.controlcenter.ksql.url`` property to the KSQL Server hostname

I'm assuming this 'url' property would require more than just the hostname? It's probably the full `http://some_host_name:8088`?